src/pages/landing/ebooks/components/_get-ebook.tsx

```diff
@@ -340,11 +338,12 @@ const GetEbook = ({ color = 'var(--color-white)', ebook_utm_code, onSubmit }: Ge
                         color={color}
                     />
                     <EmailButton
-                        isChecked={is_checked}
                         id="dm-ebook-download-signup"
                         type="submit"
                         secondary="true"
-                        disabled={is_submitting || !is_checked || email_error_msg || !email}
+                        disabled={
+                            (is_submitting || !is_checked || email_error_msg || !email) && true
```

Comment:
why is && true added ? wouldn't the logic be same without it as well.

Reply:
`disabled` requires boolean value. `since email_error_msg` is string we had a TS error here.